apps/server/src/shared/domain/entity/external-tools/external-tool/config/lti-message-type.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum LtiMessageType {
+	BASIC_LTI_LAUNCH_REQUEST = 'basic-lti-launch-request',
```

Comment:
I think there was one more in the shd

Reply:
added two other  types, which are implemented in SHD, think the other types are for LTI standard 2.0